src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -322,6 +324,12 @@ private ZipFileProcessingResult processZipFileContent(
             );
             handleInvalidFileError(Event.FILE_VALIDATION_FAILURE, containerName, zipFilename, leaseId, ex);
             return null;
+        } catch (ServiceDisabledException ex) {
+            log.error(
+                "Rejected file {} from container {} - Service is disabled", zipFilename, containerName
+            );
+            handleInvalidFileError(Event.SERVICE_DISABLED, containerName, zipFilename, leaseId, ex);
```

Comment:
Makes sense to rename this method to 'handleFileUploadError' or something like this - not 100% sure

Reply:
This method is used for all zip file failure types. I prefer to leave as it is now